scripts/install-base-packages.sh

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env bash
+# This script updates packages in the base Docker image that's used by both the
+# build and runtime images, and gives us a place to install additional
+# system-level packages with apt-get.
+#
+# Based on the blog post:
+# https://pythonspeed.com/articles/system-packages-docker/
+
+# Bash "strict mode", to help catch problems and bugs in the shell
+# script. Every bash script you write should include this. See
+# http://redsymbol.net/articles/unofficial-bash-strict-mode/ for details.
+set -euo pipefail
+
+# Display each command as it's run.
+set -x
+
+# Tell apt-get we're never going to be able to give manual feedback.
+export DEBIAN_FRONTEND=noninteractive
+
+# Refresh package lists.
+
+apt-get update
+
+# Emacs-nox and vim should cover most people's editing habits.
+# Everything uses git to determine versions and to, in the case of the
+#  cloner, do its real job.
+# Psmisc gives us a bunch of useful tools, most notably "killall".
+# Quota is helpful when debugging user filesystem issues.  The most common
+#  problem is that the user has exceeded their quota.
+# Rsync is invaluable for restarting huge copies, which is something we
+#  have to do when migrating service.
+# Screen and tmux so that you can easily have multiple shells open at
+#  once.  My muscle memory is stuck on screen but tmux is much less obscure to
+#  configure.
+# Sudo gives us the option of starting as a non-privileged user and becoming
+#  one.
```

Comment:
How? You're not installing any sudo rules. Am I missing something? (I still don't think we should ever include sudo in a container.)

Reply:
The comment's wrong.  The reason I had it in there wasn't that, it was so that you could start as root and easily become someone else, which I had thought might be useful for filesystem work, but as it turns out I've only ever done anything as root.

Reply:
Ah! Yes, that is useful, but I think for that use case you can just use `runuser`, which is much simpler and not setuid.

Reply:
So, yeah, I'll just drop it.